docs/README.md

```diff
@@ -359,25 +359,37 @@ When using a yaml configuration file, which is already available in application
 sip.core.tracing.trace-type | Sets how tracer should behave | String | "*" |
-sip.core.tracing.exchange-formatter.{property-name} | Sets value for specific property in ExchangeFormatter | / | / |
-sip.testkit.enabled | Enables SIP Test Kit | boolean | true |
+sip.core.metrics.external-endpoint-health-check.enabled | Enable health status calculation | boolean | true |
+sip.core.metrics.external-endpoint-health-check.scheduler.fixed-delay | Sets health check execution interval | Integer | 900000 |
+sip.core.metrics.external-endpoint-health-check.scheduler.initial-delay | Sets health check execution initial delay | Integer | 5000 |
+management.info.camel.enabled | Enable basic camel info under /actuator/info endpoint | boolean | false |
+management.endpoints.web.exposure.include | Set which endpoints are included | String | health,info,metrics,loggers,prometheus,adapter-routes |
 management.endpoint.health.show-details | Enable health details in actuator | String | always |
-management.endpoints.web.exposure.include | Set which endpoints are included | String | health,info,metrics,loggers,prometheus |
 springdoc.show-actuator | Show actuator API in Swagger docs| boolean | true |
 springdoc.api-docs.path | Custom path to API docs | String | /api-docs |
 springdoc.swagger-ui.path | Custom path to Swagger | String | /swagger-ui.html |
 springdoc.swagger-ui.disable-swagger-default-url | Disables default petstore API in swagger | boolean | true |
-springdoc.api-docs.enabled | Enable/Disable API docs | boolean | true |
-springdoc.swagger-ui.enabled | Enable/Disable swagger | boolean | true |
-logging.level.root | Sets the default log level | String | INFO |
+sip.testkit.enabled | Enable SIP testkit | boolean | true |
+sip.testkit.test-cases-path | Define path for file with test cases | String | test-case-definition.yml
+sip.security.ssl.enabled | Enable SIP SSL security | boolean | false |
+sip.security.ssl.server.client-auth | Enable authentication type - Possible values: NONE, WANT or NEED | String | none
+sip.security.ssl.server.key-store | Location of keystore | String | / |
+sip.security.ssl.server.key-store-password | Password of keystore | String | /
+sip.security.ssl.server.key-store-type | Type of keystore file | String | / |
+sip.security.ssl.server.key-alias | The alias (or name) under which the key is stored in the keystore | String | / |
+sip.security.ssl.server.key-password | Password of the key | String | / | 
+sip.security.ssl.client.enabled | Enable separate client certification | boolean | false |
+sip.security.ssl.client.key-store | Location of client keystore | String | /
+sip.security.ssl.client.key-store-password | Password of client keystore | String | /
+sip.security.ssl.client.key-store-type | Type of client keystore file | String | / |
```

Comment:
Can we put examples here from:
https://github.com/IKOR-GmbH/sip-framework/blob/develop/docs/security.md